wandb/sdk/launch/create_job.py

```diff
@@ -109,15 +110,20 @@ def _create_job(
 
     aliases = aliases or []
     tempdir = tempfile.TemporaryDirectory()
-    metadata, requirements = _make_metadata_for_partial_job(
-        job_type=job_type,
-        tempdir=tempdir,
-        git_hash=git_hash,
-        runtime=runtime,
-        path=path,
-        entrypoint=entrypoint,
-    )
-    if not metadata:
+    try:
+        metadata, requirements = _make_metadata_for_partial_job(
+            job_type=job_type,
+            tempdir=tempdir,
+            git_hash=git_hash,
+            runtime=runtime,
+            path=path,
+            entrypoint=entrypoint,
+        )
+        if not metadata:
+            return None, "", []
+    except Exception as e:
+        e_traceback_str = "".join(traceback.format_tb(e.__traceback__))
+        wandb.termerror(f"Error creating job: {e}/n{e_traceback_str} ")
```

Comment:
typo? is that supposed to be a newline?

Reply:
Yeah I'm actually going to remove this traceback it's just for capturing git errors and adds a lot of noise